### ISSUE TITLE
[docs] example code for static style property results in syntax error

### ISSUE THREAD

**Issue by anneb** [Polymer/lit-element#584]
The example code in the documentation for static styles on https://github.com/Polymer/lit-element/blob/master/docs/_guide/styles.md currently reads:

```
class MyElement extends LitElement {
  static styles = css`
    :host {
      display: block;
    }
  `;
}
```

This code produces a `syntax error: unexpected token '='` when used with poymer-cli (to be used as described in the same documentation under 'getting started')

**Comment by justinfagnani** [comment]
Fixed by #585

**Comment by Victor-Bernabe** [comment]
> The example code in the documentation for static styles on https://github.com/Polymer/lit-element/blob/master/docs/_guide/styles.md currently reads:
> 
> ```
> class MyElement extends LitElement {
>   static styles = css`
>     :host {
>       display: block;
>     }
>   `;
> }
> ```
> This code produces a `syntax error: unexpected token '='` when used with poymer-cli (to be used as described in the same documentation under 'getting started')

Use:


    static get styles(){
      return css`
        :host {
          display: block;
        }`;
    }
